db/migrate/20190307131832_removing_authentication_for_container_deployments.rb

```diff
@@ -0,0 +1,18 @@
+class RemovingAuthenticationForContainerDeployments < ActiveRecord::Migration[5.0]
+  class Authentication < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    say_with_time("Removing Authentications") do
+      Authentication.where(:type => 'AuthenticationAllowAll').delete_all
+      Authentication.where(:type => 'AuthenticationGithub').delete_all
+      Authentication.where(:type => 'AuthenticationGoogle').delete_all
+      Authentication.where(:type => 'AuthenticationHtpasswd').delete_all
+      Authentication.where(:type => 'AuthenticationLdap').delete_all
+      Authentication.where(:type => 'AuthenticationOpenId').delete_all
+      Authentication.where(:type => 'AuthenticationRequestHeader').delete_all
+      Authentication.where(:type => 'AuthenticationRhsm').delete_all
```

Comment:
Were these authentications only used for container deployments?  I'm assuming yes but perhaps `AuthenticationGoogle` is used for the google provider?  Maybe?  I have no idea.

Reply:
@agrare do you know if these auth types can be removed?

Reply:
These were all added for container deployments, https://github.com/ManageIQ/manageiq/commit/6a58d23b83cdac71b619d58c6956d393103f939f

Checking to see if they're in use anywhere else...

Reply:
I don't see these in use anywhere besides container_deployment currently, good to go from me